libkpod/common/common.go

```diff
@@ -10,6 +10,11 @@ import (
 	"github.com/containers/image/types"
 )
 
+var (
+	// ErrNoPassword is returned if the user did not supply a password
+	ErrNoPassword = errors.New("password was not supplied")
```

Comment:
Can we make this a wrapped error
errors.Wrapf(syscall.EINVAL, password was not supplied")